addons/delivery/i18n/delivery.pot

```diff
@@ -655,7 +655,7 @@ msgstr ""
 #. module: delivery
 #: code:addons/delivery/models/delivery_grid.py:0
 #, python-format
-msgid "No price rule matching this order; delivery cost cannot be computed."
+msgid "Not available for current order"
```

Comment:
should be below "Normal Delivery Charges" I think (but in master I think it's not necessary)